test/snapshots/test/fixtures/skip-test/repro.test.ts

```diff
@@ -0,0 +1,19 @@
+import { describe, expect, it } from 'vitest'
+
+const ENABLE_SKIP = process.env.ENABLE_SKIP;
+
+describe.skipIf(ENABLE_SKIP)('repro suite', () => {
+  it('inner case', () => {
+    expect('hi-1').toMatchSnapshot()
+  })
+})
+
+it.skipIf(ENABLE_SKIP)('top-level case', () => {
+  expect('hi-2').toMatchSnapshot()
+})
+
+// requires at least one non-skipped test to trigger
+// `SnapshotClient.startCurrentRun` on current file
+it('normal case', () => {
```

Comment:
This looks like an unfortunate edge case. I think this PR should also cover files that have only a single `describe.skipIf`

Reply:
> I think this PR should also cover files that have only a single `describe.skipIf`

Sorry, the comment here might be confusing, but the reason why I have this is to reproduce the bug. Without this "normal case", the old code looks working and new code is also working (so this is for 2nd checkbox, "include a test that fails without this PR but passes with it.")

Does this make sense? Or did mean something else?

Reply:
Do you mean that without this `it`, the code works before and after? But with this `it` the code didn't work before?

Reply:
Yes, that's what I mean. I adjusted my comment a bit https://github.com/vitest-dev/vitest/pull/4795/commits/4b480ef0f0c73b257dcc3c442e3579a94dcb4d23

Reply:
Ok then 👍🏻